go.mod

```diff
@@ -1,16 +1,18 @@
 module github.com/ipfs/interface-go-ipfs-core
 
 require (
-	github.com/ipfs/go-cid v0.0.3
-	github.com/ipfs/go-ipfs-files v0.0.2
-	github.com/ipfs/go-ipld-cbor v0.0.1
-	github.com/ipfs/go-ipld-format v0.0.1
-	github.com/ipfs/go-merkledag v0.0.3
-	github.com/ipfs/go-path v0.0.3
-	github.com/ipfs/go-unixfs v0.0.4
-	github.com/libp2p/go-libp2p-core v0.2.2
-	github.com/multiformats/go-multiaddr v0.0.4
-	github.com/multiformats/go-multihash v0.0.7
+	github.com/ipfs/go-cid v0.0.6
+	github.com/ipfs/go-ipfs-files v0.0.8
+	github.com/ipfs/go-ipld-cbor v0.0.4
+	github.com/ipfs/go-ipld-format v0.2.0
+	github.com/ipfs/go-merkledag v0.3.2
+	github.com/ipfs/go-path v0.0.7
+	github.com/ipfs/go-unixfs v0.2.4
+	github.com/libp2p/go-libp2p-core v0.6.0
+	github.com/multiformats/go-multiaddr v0.2.2
+	github.com/multiformats/go-multibase v0.0.3
+	github.com/multiformats/go-multihash v0.0.13
+	golang.org/x/crypto v0.0.0-20200510223506-06a226fb4e37
```

Comment:
could you modify the first commit with a `go mod tidy` and repush. This crypto dependency is indirect at best.